Makefile

```diff
@@ -50,9 +57,9 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 	$(Q)$(CC) -o $(OUT)/rom2h tests/rom2h.c
 	@$(OUT)/rom2h
 
-$(OUT)/bench: $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
+$(OUT)/bench: gbit/lib/tester.c gbit/lib/ref_cpu.c gbit/lib/inputstate.c gbit/lib/disassembler.c
```

Comment:
Add the following statement:
```
GBIT_OBJS = \
    gbit/lib/tester.o \
    gbit/lib/ref_cpu.o \
    gbit/lib/inputstate.o \
    gbit/lib/disassembler.o
```

And then, specify the dependency of `bench` to `$(GBIT_OBJS)`. You have to ensure to clean the object files in `clean` target.